src/raphael_backend_flask/process.py

```diff
@@ -0,0 +1,64 @@
+import json
+from typing import Generator
+
+import requests
+
+from health_misinfo_shared.vertex import process_video
+from raphael_backend_flask.db import execute_sql
+from raphael_backend_flask.youtube import download_captions, extract_title
+
+
+def update_video_transcript(
+    video_id: str,
+    **kwargs,
+) -> None:
+    fields = ", ".join(kwargs.keys())
+    entries = ", ?" * len(kwargs)
+
+    execute_sql(
+        f"REPLACE INTO video_transcripts (id, {fields}) VALUES (?{entries})",
+        (
+            video_id,
+            *kwargs.values(),
+        ),
+    )
```

Comment:
This bit is so ugly – I am sorry. I want a way to only update the relevant columns. This works, but I don’t like it.